cmd/kubelet/app/server.go

```diff
@@ -202,11 +204,25 @@ is checked every 20 seconds (also configurable with a flag).`,
 			}
 
 			// load kubelet config file, if provided
-			if configFile := kubeletFlags.KubeletConfigFile; len(configFile) > 0 {
+			configFile := kubeletFlags.KubeletConfigFile
+			if len(configFile) > 0 {
 				kubeletConfig, err = loadConfigFile(configFile)
 				if err != nil {
 					return fmt.Errorf("failed to load kubelet config file, error: %w, path: %s", err, configFile)
```

Comment:
error is typically goes last in the string. It's easier to understand the error this way